packages/poi-preset-karma/index.js

```diff
@@ -76,6 +71,36 @@ module.exports = (options = {}) => {
 
       delete webpackConfig.entry
 
+      if (coverage) {
+        /* for general usage */
+        webpackConfig.module.rules = [
+          {
+            test: /\.(jsx?)$/,
+            exclude: /(node_modules|\.test\.jsx?)/,
+            enforce: 'post',
+            loader: 'istanbul-instrumenter-loader',
+            query: {
+              esModules: true
+            }
+          }
+        ].concat(webpackConfig.module.rules)
+        /* for vue (assumes vue-loader) */
+        webpackConfig.module.rules = webpackConfig.module.rules
```

Comment:
actually you can update webpack config in `poi.extendWebpack('test', config => {})` using webpack-chain

Reply:
I will update this :)

Reply:
Thanks. I'm new to `webpack-chain` will check it out. It looks promising.